plugins/nf-google/src/main/nextflow/cloud/google/lifesciences/GoogleLifeSciencesHelper.groovy

```diff
@@ -176,6 +176,9 @@ class GoogleLifeSciencesHelper {
         disk.setSizeGb(req.diskSizeGb)
 
         def serviceAccount = new ServiceAccount().setScopes( [SCOPE_CLOUD_PLATFORM] )
```

Comment:
not sure if we want/need to specify customized scope. But let me know if we should. 
If yes, what would it be? List of String? 

Thanks

Reply:
What's the use of the case of custom scope?

Reply:
I can not think of any use case that require custom scope. Seems "cloud-platform API scope" already added by default.  One possible use case might be the batch job need to access other Google APIs besides GCP(e.g. gsuite, drive)? 
That does feel rate, so I did not add support for custom scope in this PR.

Reply:
Yeah there isn't really any good reason to modify the scope.  Historically this was useful since there was a specific scope for the Genomics API (predecessor to Life Sciences), and cloud-platform doesn't grant *everything*, but over time most APIs are converging on it and using IAM to control access on a finer grained basis.

Reply:
Fair enough, therefore the scope setting can be ignored